### PR TITLE
[14.0] [MIG] sale_order_price_recalculation

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py [hunk 1]
```diff
@@ -78,7 +78,8 @@ def test_price_recalculation(self):
         self.sale_order_line.name = "My product description"
         self.assertEqual(self.sale_order_line.price_unit, self.product.lst_price)
         # Change price
-        self.product.lst_price = 500
+        with Form(self.product) as product:
+            product.lst_price = 500
```
Comment (legalsylvain) [inline sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py:82]: naive question. Why this change ?

**Review by legalsylvain — COMMENTED** [review]
1 question, LGTM otherwise.

File: sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py:82]: Direct price set was not calling price_compute of product at the time of recalculation, hence price not set to 500 in sale order line and test failing

**Review by MiquelRForgeFlow — APPROVED** [review]

File: sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py (hunk 1 above)
Comment (legalsylvain) [reply sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py:82]: oki, thanks

**Review by legalsylvain — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Comment by andreampiovesana** [comment]
merge?
